HHS/Head-Start-TTADP
# Filter by program specialist, display all reasons in reason widget, update ignored yarn audit

## Description of change
- Both alert and report grid views on the AR landing should now have a filter for Program Specialist.
- The reason list widget displays all reasons, not a subset of 13.
- Yarn audit ignored vulnerabilities have been updated

## How to test
- The Program specialist filter should work
- All reasons are displayed, not a smaller set (on staging, change date range to 09/01/2020-09/23/2021)
- The CI should pass

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-246
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-355


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
